### DISCUSSION TITLE
Having trouble using vkh.h

### DISCUSSION THREAD

**Discussion by notshriram** [jpbruyere/vkvg#124] [Q&A] [answered]
I am on macOS 12.4 M1. I have vulkan SDK (MoltenVK) installed.

I did the following steps after cloning the repository with the --recursive flag
```bash
mkdir build && cd build
cmake ..
make
make install
```

whenever I try to use the library, I get the following error
```
/usr/local/include/vkh.h:33:10: fatal error: 'deps/tinycthread.h' file not found
#include "deps/tinycthread.h"
         ^~~~~~~~~~~~~~~~~~~~
1 error generated.
```

**Comment by jpbruyere** [comment, marked as answer]
Thanks for the report, this is corrected in latest master. see #125 .
